code/development-env/config/ca/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.crt
```

Comment:
It is great that all the docs to generate certs etc are all here, but is there any harm in committing them so we don't all have to generate our own?

Reply:
They are development certificates and therefore can be freely distributed. I can the certificates I have generated. Will update.